tests/Unit/Collections/CollectionTestCase.php

```diff
@@ -33,17 +35,17 @@ abstract class CollectionTestCase extends TerminusTestCase
     /**
      * @return Config
      */
-    public function getConfig()
+    public function getConfig(): TerminusConfig
     {
         return $this->config;
     }
 
     /**
      * @inheritdoc
      */
-    protected function setUp()
+    protected function setUp(): void
     {
-        $this->config = $this->getMockBuilder(Config::class)
+        $this->config = $this->getMockBuilder(TerminusConfig::class)
```

Comment:
Can we start fresh with unit tests and not copy the existing over-use of mocks? Why mock a config object when we can build a real one, for example.